client/hawc_client/summary.py

```diff
@@ -73,7 +73,7 @@ def get_visual(self, visual_id: int):
             visual_id (int): ID of the visual to read
 
         Returns:
-            dict: The result object, if get was successful
```

Comment:
unlike other `get_<model>` functions in the client, visual and datapivot return the resopnse object instead of a dict. Should I make it consistent and return the dict here? Seems like an oversight

Reply:
great idea - let's add a jira card and do next quarter, or after the testing work is complete!